src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/induction/PreviousTrainingEntity.kt

```diff
@@ -0,0 +1,79 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.induction
+
+import jakarta.persistence.CollectionTable
+import jakarta.persistence.Column
+import jakarta.persistence.ElementCollection
+import jakarta.persistence.Entity
+import jakarta.persistence.EntityListeners
+import jakarta.persistence.EnumType
+import jakarta.persistence.Enumerated
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import jakarta.validation.constraints.NotNull
+import org.hibernate.annotations.CreationTimestamp
+import org.hibernate.annotations.UpdateTimestamp
+import org.hibernate.annotations.UuidGenerator
+import org.springframework.data.annotation.CreatedBy
+import org.springframework.data.annotation.LastModifiedBy
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
+import java.time.Instant
+import java.util.UUID
+
+/**
+ * Holds details of any additional training that a Prisoner may have done.
+ *
+ * Note that the list of training cannot be empty, since NONE is an option.
+ */
+@Table(name = "previous_training")
+@Entity
+@EntityListeners(AuditingEntityListener::class)
+class PreviousTrainingEntity(
+  @Id
+  @GeneratedValue
+  @UuidGenerator
+  var id: UUID? = null,
+
+  @Column(updatable = false)
+  @field:NotNull
+  var reference: UUID? = null,
+
+  @ElementCollection(targetClass = TrainingType::class)
+  @Enumerated(value = EnumType.STRING)
+  @CollectionTable(name = "training_type")
```

Comment:
This is how I'm representing a list of enums. I've distinguished these "collection tables" in the comments in the Flyway script.